include/linux/slab.h

```diff
@@ -113,10 +113,9 @@
 #endif
 
 #ifdef CONFIG_KFENCE
-#ifdef CONFIG_KASAN
-#error Conflicting definitions
-#endif
 #define SLAB_KFENCE		((slab_flags_t __force)0x08000000U)
```

Comment:
We probably need a comment that this flag clashes with SLAB_KASAN.

Reply:
Can add comment. Kconfig 'depends on !KASAN' takes care of enforcing this though.

Reply:
That's true, but people looking at slab.h may not know KFENCE doesn't work together with KASAN